codegen/src/api/runtime_apis.rs

```diff
@@ -11,6 +11,48 @@ use proc_macro2::TokenStream as TokenStream2;
 use quote::{format_ident, quote};
 
 /// Generates runtime functions for the given API metadata.
+///
+/// # Note
```

Comment:
This note is the sort of thing that could fall out of date fairly quickly, so personally I'd just be looking at the generated code anyway to see how things are structured and would avoid such a comment :)